packages/components/organisms/f-cookie-banner/test-utils/component-objects/f-cookie-banner-legacy.component.js

```diff
@@ -9,15 +9,15 @@ module.exports = class CookieBannerLegacy extends Page {
 
     get cookiePolicyLink () { return $('[data-test-id="cookie-policy-link"]'); }
 
-    get cookiePolicyLinkHref () { return this.cookiePolicyLink.getAttribute('href'); }
-
     get closeButton () { return $('[data-test-id="cookieBanner-close-button"]'); }
 
-    clickCookiePolicyLink () { this.cookiePolicyLink.click(); }
+    async cookiePolicyLinkHref () { return this.cookiePolicyLink.getAttribute('href'); }
```

Comment:
Changed this to a function due to `getAttribute` being a WDIO async function, so can no longer be a getter.